JSDemos/Demos/Popup/Scrolling/jQuery/index.html

```diff
@@ -13,6 +13,44 @@
     <script src="index.js"></script>
   </head>
   <body class="dx-viewport">
-    <div id="popup"></div>
+    <div class="demo-container">
+      <div class="button-container">
+        <div id="show-scrollable-popup"></div>
```

Comment:
We discussed the naming with Vlada. 
Let's use the following naming: just a "popup" and a "popupWithScrollView". 
I also ask u to make the naming consistent across the frameworks.

Points:
- "scrollViewPopup" is not very good from a semantic point view
- "scrollable" and so on is a long, similar to "scrollable" component from docs
- we wanna emphasize the default behavior

```suggestion
        <div id="show-popup-button"></div>
```

Let's also ask Vlada for a final review after all changes

Reply:
Renamed in [fix notes after review](https://github.com/DevExpress/devextreme-demos/pull/2486/commits/0f1b08d8909b276906184e23e235d9bfc1f08533). @vladaskorohodova  check it please again

Reply:
Fixed